### PR TITLE
Roles

### PR DESCRIPTION
**Pull request by jarrydlee**
fixes https://github.com/PagerNation/PagerNation/issues/131

### CODE REVIEW

File: server/tests/routes/user.test.js [hunk 1]
```diff
@@ -166,6 +167,50 @@ describe('## User APIs', () => {
...
+    context('with an admin user', () => {
+      before((done) => {
+        buildAndAuth('user', fixtures.user({ isSysAdmin: true }))
+          .then((u) => {
+            userAndToken = u;
+            done();
+          });
+      });
+
+      it('should update a users role', (done) => {
```
Comment (kalebdavis) [inline server/tests/routes/user.test.js]: nit: apostrophe

File: server/services/user.js [hunk 2]
```diff
@@ -10,10 +10,15 @@ const DEVICE_SCHEMA = Joi.object().keys({
...
+function exists(userId) {
+  return User.exists(userId);
+}
+
 function createUser(userObject) {
   const userSchema = Joi.object().keys({
     name: Joi.string().required(),
-    email: Joi.string().email().required()
+    email: Joi.string().email().required(),
+    isSysAdmin: Joi.boolean()
```
Comment (kalebdavis) [inline server/services/user.js]: Does this mean that anyone can add a sysAdmin?

Comment (jarrydlee) [inline server/services/user.js]: Nope, it just means that the updateUser method can be used to make that update. There is middleware that stops non admins from creating admins

Comment (kalebdavis) [inline server/services/user.js]: But this change is added to `createUser` not `updateUser`...where's the middleware stopping that? I see it on `updateUser` but not `create`

Comment (jarrydlee) [inline server/services/user.js]: Missed that, yeah it's still behind validation which doesn't allow for the isSysAdmin field to be passed in through the api

**Pull request merged**